src/site/artifacts.rs

```diff
@@ -20,15 +20,15 @@ fn get_kind_string(kind: &ArtifactKind) -> String {
     }
 }
 
-fn create_download_link(config: &Config, name: &String) -> String {
+fn build_downloadlink(config: &Config, name: &str) -> String {
     if let (Some(repo), Some(version)) = (&config.repository, &config.version) {
         format!("{}/releases/download/v{}/{}", repo, version, name)
     } else {
         String::new()
     }
 }
 
-pub fn create_artifacts_header(config: &Config) -> Result<Option<Box<div<String>>>> {
+pub fn build_section(config: &Config) -> Result<Option<Box<div<String>>>> {
```

Comment:
Maybe "build_artifacts_section" no?

Reply:
it's in the artifacts module so i'm trying to not repeat the name, since you'll be calling it `artifacts::build_section`